pages/common/pushd.md

```diff
@@ -1,7 +1,7 @@
 # pushd
 
 > Place a directory on a stack so it can be accessed later.
-> See also `popd` to switch back to original directory.
+> See also `popd` to switch back to original directory and, on UNIX-like operating systems, `dirs` to display directory stack contents.
```

Comment:
```suggestion
> See also `popd` to switch back to original directory, and `dirs` to display directory stack contents.
```

Reply:
I added that with the windows user in mind: I could see someone running `tldr pushd`, following up with `tldr dirs`, and then getting frustrated when they try to use the nonexistent-on-windows `dirs` command. But I'm happy to go with your judgement here if you think the simpler description is better overall.

Reply:
Right, I agree.  But since we already have Windows pages for [`pushd`](https://github.com/tldr-pages/tldr/blob/de637c416d65d4b849e01677caad1faea3e435bc/pages/windows/pushd.md) and [`popd`](https://github.com/tldr-pages/tldr/blob/de637c416d65d4b849e01677caad1faea3e435bc/pages/windows/popd.md), clients on Windows will display that page rather than the ones in `common/`, so they won't even see the pointer to `dirs`.